1-estrategia/README.md

```diff
@@ -2,22 +2,129 @@
+
 ### Indicadores de éxito
 
+* Happiness (Satisfacción de los usuarios en relación con el producto): Se quiere la satisfacción del usuario mediante una valoración que realizarán ellos mismos al momento de  abandonar la plataforma. Es decir un tipo de encuesta donde 1 es insatisfecho y 5 es muy satisfecho + un box de sugerencias.
+
+* Engagement (Grado de respuesta del usuario sobre el producto): “Para medir el grado de recomendación que realizan los usuarios de nuestro sitio web podemos utilizar el criterio de clasificación denominado NPS (Net Promoter Score), para lo cual se clasifican en tres grupos: Los detractores serían aquellos con una puntuación de entre 0 y 6, los pasivos aquellos que nos otorguen entre un 7 y un 8 y los promotores serían aquellos que puntuaron entre un 9 y un 10 dicha sección de nuestra web”.→ Para nuestro sitio, podemos utilizar este criterio de clasificación para medir el grado de recomendación de unos inmigrantes a otros. Es muy interesante de utilizar este NPS para nuestro sitio porque podemos saber los sentimientos que tienen las personas por el sitio web.
+
+
+* Adoption (El número de usuarios que se registra en un determinado espacio temporal): Es necesario establecer un objetivo de usuarios nuevos en el sitio web vinculado a un porcentaje (ejemplo 90%) de inmigrantes forzados nuevos que arriben al País Vasco mensualmente.
+
+* Retention (El número de usuarios que utiliza un producto durante un determinado tiempo): Es posible que el usuario encuentre diversas dificultades en el sitio web, por esta razón se construyen indicadores como: 
+Ratio de usuarios que han completado todas las tareas
+Ratio de usuarios que han cometido algún error en alguna de las tareas
+Cantidad media de errores cometidos por los usuarios durante las tareas realizadas
+Tiempo estimado para la realización de cada tarea
+Grado de dificultad encontrado para completar cada tarea
+
+* Task success (Grado de satisfacción respecto a la consecución de las tareas que permite la web): Podemos cuestionar a los usuarios de nuestro sitio web para tener un mejor visión de cómo los usuarios vean nuestro sitio web. Es necesario de conocer la idea del usuario sobre nuestro sitio web.
+
 ## Necesidades del usuario
+Segmentación de usuarios
+Generales → Hombre y mujeres 20-45 años
+Específico → Hombres y mujeres entre 20 y 45 años de clase media-baja, que provienen de centroamérica, norte de sudamérica y marruecos.
 
 ### Investigación de usuarios
```

Comment:
¿Habéis realizado esta encuesta, aunque sea a 2-3 personas? ¿Qué conclusiones podéis extraer de esas entrevistas?